hypothesis-python/tests/cover/test_lambda_formatting.py

```diff
@@ -64,6 +68,9 @@ def test_does_not_error_on_unparsable_source():
     assert get_pretty_function_description(t) == "lambda x: x"
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 11), reason="nested lambdas fail compile-test"
+)
```

Comment:
I'm curious what changed in this PR, since master passes these tests on 3.10, right?

Reply:
> If it turns out to be a performance hotspot, I would be fine with keeping the previous behavior of no compile-test if only one lambda, since that only helps when editing files live, IIUC.

I don't think it is. We could keep open the possibility to revert this behaviour if it becomes too difficult to keep the compile-test robust, though. Does that warrant a code comment or will we remember?